x/gamm/types/constants.go

```diff
@@ -15,6 +15,9 @@ const (
 	BalancerGasFeeForSwap = 10_000
 
 	StableswapMaxScaledAmtPerAsset = 10_000_000_000
+	StableswapMinScaledAmtPerAsset = 1
+	// TODO: increase this to 1_000_000 (aligned with 6 decimal place tokens) and update tests accordingly
```

Comment:
Note: since most of our tests currently use < 1_000_000 tokens per asset, most of them would need to be refactored after this change